README.md

```diff
@@ -34,6 +34,9 @@ Check out [the docs](http://hexdocs.pm/ex_machina/ExMachina.html) for more detai
 Define factories:
 
 ```elixir
+# You could put this in `test/support/factory.ex` if you use it for tests only,
```

Comment:
Hm, seems the default `elixirc_paths` is just `lib`. I tried adding an .ex file to `test` in a new project (non-Phoenix) and it wasn't available to use from tests.

So I guess if you've used this outside Phoenix, you made sure to add whatever directory it was to `elixirc_paths`? If so we could doc that.